.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,76 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: "42 3 * * *"
```

Comment:
Didn't even know that feature! Why not just doing it `@monthly`?

Reply:
Ide tells me 

![image](https://user-images.githubusercontent.com/327717/104817295-b2fc2980-5820-11eb-8ebe-8f3053364aa0.png)

can use it tho if you with

Reply:
try if this works, otherwise use `0 0 1 * *`.

Reply:
invalid `cron` attribute "@monthly" 

used `0 0 1 * *`